src/components/PrimaryPanes/Outline.js

```diff
@@ -56,10 +56,40 @@ export class Outline extends Component {
     );
   }
 
-  renderFunctions(symbols: Array<SymbolDeclaration>) {
+  renderClassFunctions(functions: SymbolDeclaration[]) {
+    const classFunctions = functions.filter(
+      func => func.name != "anonymous" && !!func.klass
+    );
+
+    if (classFunctions.length == 0) {
+      return null;
+    }
+
+    const klass = classFunctions[0].klass;
+    const klassFunc = find(functions, func => func.name === klass);
```

Comment:
Could `Array::find` work here as well, @jasonLaster ? We seem to have mixed use of lodash and native find across the code.

Reply:
Yeah, that's a good find! that would be a nice PR because we have good test coverage here